scss/civicrm/contact/pages/_contributions.scss

```diff
@@ -746,8 +746,8 @@
 
 #{civi-page('contact-view')} .ui-dialog .CRM_Contribute_Form_Contribution {
   .crm-link-credit-card-mode {
-    position: absolute;
-    right: 0;
+    float: right;
+    margin-bottom: -30px !important;
```

Comment:
magic number, why?

Reply:
This is to make a button proper 
Before
![image](https://user-images.githubusercontent.com/26058635/37821581-0967bb34-2eaa-11e8-8bf0-86b649d0a551.png)

After
![image](https://user-images.githubusercontent.com/26058635/37821600-159613ec-2eaa-11e8-8c30-7bf203b3e278.png)